chains/evm/calls/bridge/bridge.go

```diff
@@ -0,0 +1,286 @@
+package bridge
+
+import (
+	"bytes"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/client"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/consts"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/contract"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/transactor"
+	"math/big"
+	"strings"
+
+	"github.com/ChainSafe/chainbridge-core/chains/evm/voter/proposal"
+	"github.com/ChainSafe/chainbridge-core/relayer/message"
+	"github.com/ChainSafe/chainbridge-core/types"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/rs/zerolog/log"
+)
+
+type BridgeContract struct {
+	contract.Contract
+}
+
+func NewBridgeContract(
+	client client.ContractCallerDispatcherClient,
+	bridgeContractAddress common.Address,
+	transactor transactor.Transactor,
+) *BridgeContract {
+	a, err := abi.JSON(strings.NewReader(consts.BridgeABI))
```

Comment:
We can probably just have abi as a var and assign abi.JSON directly to that
There is not really a chance that it will error.